_schemaV2.graphql

```diff
@@ -20091,6 +20094,8 @@ interface Sellable {
   saleMessage: String
 }
 
+union SellerType = Partner
```

Comment:
Making the seller type a union, aligned with Exchange assumption, to (hopefully) avoid breaking changes when seller can be other than partners.